armi/bookkeeping/db/passiveDBLoadPlugin.py

```diff
@@ -0,0 +1,104 @@
+
+This plugin can allow you to more easily open a database, because you can ignore sections of the
+blueprint files, and ignore any parameters as you want.
+
+This was designed to allow loading an ARMI database without the application that created it.
+"""
+import yamlize
+
+from armi import plugins
+from armi.reactor import parameters
+from armi.reactor.parameters import ParamLocation
+from armi.utils import units
+
+
+class PassThroughYamlize(yamlize.Object):
+    """Just a helper for PassiveDBLoadPlugin, to allow for ignore unknown blueprints sections."""
+
+    @classmethod
+    def from_yaml(cls, loader, node, round_trip_data=None):
+        return yamlize.Object.from_yaml.__func__(
+            PassThroughYamlize, loader, node, round_trip_data
+        )
+
+
+class PassiveDBLoadPlugin(plugins.ArmiPlugin):
+    """Provides the ability to ignore parameters sections of blueprint files.
+
+    This plugin allows you to more easily open a database, because you can ignore sections of the
+    blueprint files and as many of the parameters as you want.
+
+    This was designed to allow loading an ARMI database without the application that created it.
+
+    Notes
+    -----
+    Obviously, if you are ignoring huge groups of parameters or whole sections of the blueprints,
+    you are losing information. There is no way to use this plugin and still claim full fidelity of
+    your understanding of the reactor. ARMI does not support any such claims.
+    """
+
+    SKIP_BP_SECTIONS = []
+    SKIP_PARAMS = {}
+
+    @staticmethod
+    @plugins.HOOKIMPL
+    def defineBlueprintsSections():
+        """Ignore a pre-determined set of blueprint sections."""
+        skips = []
+        for skippedBp in PassiveDBLoadPlugin.SKIP_BP_SECTIONS:
+            skips.append(
+                (
+                    skippedBp.replace(" ", ""),
+                    yamlize.Attribute(
+                        key=skippedBp, type=PassThroughYamlize, default=None
+                    ),
+                    PassThroughYamlize,
+                )
+            )
+
+        return skips
+
+    @staticmethod
+    @plugins.HOOKIMPL
+    def defineParameters():
+        """Define parameters for the plugin."""
+        # build all the parameters we are missing in default ARMI
+        params = {}
+        for dataClass, paramNames in PassiveDBLoadPlugin.SKIP_PARAMS.items():
+            if len(paramNames):
+                params[dataClass] = PassiveDBLoadPlugin.buildParamColl(paramNames)
+
+        return params
+
+    @staticmethod
+    def buildParamColl(names):
+        """Try replacing any missing parameters with unitless nonsense."""
+        # handle the special cases
+        for nomen in ["flags", "serialNum"]:
+            if nomen in names:
+                names.remove(nomen)
+
+        # build a collection of defaulted parameters to passively ignore
+        desc = "This is just a silly placeholder Parameter. It's meaning is unknown."
```

Comment:
The case that @alexhjames brings up with `saveToDB` set to True brings up an interesting point about saving state. Imagine we have app1 that makes a database that is loaded by app2 and app2 adds new parameters and saves the database. Now opening this back into app1 would not only drop the descriptions that app2 added, but wouldn't it also drop the descriptions of the parameters that app1 should know about? 

In the most extreme case I think you could end up with unit less and no descriptions on all parameters in the database? Could be totally wrong here but his comment sparked the thought.